src/frontends/tensorflow/src/op/conv_3d.cpp

```diff
@@ -17,10 +17,14 @@ namespace op {
 OutputVector translate_conv_3d_op(const NodeContext& node) {
     auto ng_input = node.get_input(0), ng_filter = node.get_input(1);
 
+
+    // retrieve attributes for Conv3D
     auto tf_strides = node.get_attribute<std::vector<int64_t>>("strides");
-    auto tf_dilations = node.get_attribute<std::vector<int64_t>>("dilations");
     auto tf_padding_type = node.get_attribute<std::string>("padding");
-    auto tf_data_format = node.get_attribute<std::string>("data_format");
+
+    // retrieve optional attributes
+    auto tf_data_format = node.get_attribute<std::string>("data_format", "NDHWC");
+    auto tf_dilations = node.get_attribute<std::vector<int64_t>>("dilations", {1, 1, 1, 1, 1});
```

Comment:
Shouldn't default dilations for 3D case be of size 6, not 5? Same for backprop 3d case

Reply:
Okay, my bad